auth/auth.go

```diff
@@ -37,6 +37,7 @@ const (
 
 	// SDK-generated error codes
 	idTokenRevoked       = "ID_TOKEN_REVOKED"
+	idTokenDisabled      = "ID_TOKEN_DISABLED"
```

Comment:
This is called `USER_DISABLED` in other languages. Let's use the same terminology here.